README.md

```diff
@@ -14,3 +14,11 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+1. Asymptotic analysis could be misleading compared to practice in a few different ways. Firstly, just because algorithm A is more efficient asymptotically than algorithm B, it doesn't mean that with the same input A will always be more efficient that algorithm B, just that it will tend to be at large numbers. This ties into a second confusing thing, where the average-case for algorithm A may be better than algorithm B, but the best case of B may be better than A. This is like the example of comparing insertion sort's best case of n to quick sort's best case of nlogn. Quick sort is a much more efficient algorithm, but when given a already sorted array, it will confirm it is sorted more quickly than quick sort. A third misleading aspect of asymptotic analysis is that just because two algorithms share the same asymptotic complexity of say nlogn, one is still likely faster than the other. This is because when considering the asymptotic complexity of an algorithm, we are essentially considering the magnitude of its variables. For example, 3nlogn would be more efficient than 100nlogn, but they are still part of the same $\Theta(nlogn)$.
```

Comment:
No convinced by your first two reasons -- we can analyze best/worst/average case separately, and as we said in class there's no relationship between them and you can't necessarily deduce one from the other. And what about the definitions of asymptotic complexity tells you that the performance in practice might be very different?